src/javascripts/views/homeView.js

```diff
@@ -1,7 +1,85 @@
+import Template from "../templates/template";
 class HomeView {
     constructor() {
-        this.transactionListEl = document.querySelector(".transaction-list")
+        this.categoryEl = document.querySelector(".category-list");
         this.addBtnEl = document.querySelector(".btn-add-transaction");
+        this.summaryEl = document.querySelector(".summary");
+        this.showTransactionsBtnEl = document.querySelector(".transaction-title-text");
```

Comment:
Choose another name

Reply:
I fixed it @phuocduong-agilityio  . Please help me check. Thanks !